### PR TITLE
mentions added

### PR DESCRIPTION
**Pull request by Lt-Kartik-9PARA-SF**
added mentions

### CODE REVIEW

**Comment by alphainfinitus** [comment]
image

When I try to type in the username.

**Comment by alphainfinitus** [comment]
image

If a username is repeated it's not being replaced with link.

File: src/ui-components/MarkdownEditor.tsx [hunk 1]
```diff
@@ -214,6 +218,57 @@ function MarkdownEditor(props: Props): React.ReactElement {
...
+					`[@${userName}](${global.window.location.origin}/user/${userName})`
+				);
+				replacedUsernames.push(userName);
+			}
+		}
+		const res = await nextApiClientFetch(
+			`api/v1/auth/data/userProfileWithUsername?username=${usernameQuery}`
+		);
+		if (res) {
+			setLastFoundUser(res.data.username);
```
Comment (alphainfinitus) [inline src/ui-components/MarkdownEditor.tsx:245]: 'res.data' is of type 'unknown'

File: src/ui-components/MarkdownEditor.tsx [hunk 2]
```diff
@@ -214,6 +218,57 @@ function MarkdownEditor(props: Props): React.ReactElement {
 		return Boolean(url);
 	};
 
+	const [input, setInput] = useState('');
+	const [lastFoundUser, setLastFoundUser] = useState('');
+
+	async function apiCall(usernameQuery: string, content: string) {
```
Comment (alphainfinitus) [inline src/ui-components/MarkdownEditor.tsx:224]: better function name please

File: pages/_app.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Poppins, Roboto_Mono, Work_Sans } from '@next/font/google';
+import { Poppins, Roboto_Mono, Work_Sans } from 'next/font/google';
```
Comment (alphainfinitus) [inline pages/_app.tsx:5]: This'll cause build errors please revert to `@next/font/google`

Comment (Lt-Kartik-9PARA-SF) [reply pages/_app.tsx:5]: yes ill revert it and do the other bug fix as well

**Review by alphainfinitus — APPROVED** [review]

**Pull request merged**